### PR TITLE
added note i64 values in web embedded

### PR DESCRIPTION
**Pull request by wanderer**

### CODE REVIEW

File: Web.md [hunk 1]
```diff
@@ -57,6 +57,11 @@ would inhibit export. In general, `ArrayBuffer` could be viewed as an
 optimization of copy in/out that was only available when linear memory
 behaved like an `ArrayBuffer` (or `SharedArrayBuffer`).
 
+## 64-bit integers
+Currently Javascript doesn't support 64-bit itergers so they will be disallowed
```
Comment (jfbastien) [inline Web.md]: "integers"

File: Web.md [hunk 2]
```diff
@@ -57,6 +57,11 @@ would inhibit export. In general, `ArrayBuffer` could be viewed as an
 optimization of copy in/out that was only available when linear memory
 behaved like an `ArrayBuffer` (or `SharedArrayBuffer`).
 
+## 64-bit integers
+Currently Javascript doesn't support 64-bit integers so they will be disallowed
+as a return value in JS binding. Once JS has [Typed Objects](http://wiki.ecmascript.org/doku.php?id=harmony:typed_objects)
```
Comment (titzer) [inline Web.md]: I think you should explicitly mention imports and exports and parameter and return types.

File: Web.md [hunk 3]
```diff
@@ -57,6 +57,12 @@ would inhibit export. In general, `ArrayBuffer` could be viewed as an
 optimization of copy in/out that was only available when linear memory
 behaved like an `ArrayBuffer` (or `SharedArrayBuffer`).
 
+## 64-bit integers
+
+Currently Javascript doesn't support 64-bit integers so they will be disallowed
+as parameters in imports and exports and in return values in the JS bindings. Once JS has [Typed Objects](http://wiki.ecmascript.org/doku.php?id=harmony:typed_objects)
```
Comment (wanderer) [inline Web.md]: @titzer is this accurate?

Comment (titzer) [inline Web.md]: You can probably just drop the Typed Objects reference.

Comment (wanderer) [inline Web.md]: ok updated

**Comment by sunfishcode** [comment]
Also related: https://github.com/WebAssembly/design/issues/515

**Comment by flagxor** [comment]
So I believe the intention was that i64 be allowed in imports + exports, but fail if called from JS. This allows hooking up cross module calls with i64 args.

This should be clarified in the docs. Will do so soon (updated #515 to reflect this).

**Pull request closed without merging**